### PR TITLE
#5 Improve Documentation

### PR DESCRIPTION
**Pull request by jzbbrv-hpi**
Improvement to GitHub and ReadTheDocs Documentation for improved usability.

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -1,19 +1,43 @@
...
+    simba_ml.__version__
+
+For more detailed installation instructions and requirements, see :ref:`installation`.
+
+For an example usage of SimbaML, see :ref:`quickstart`.
+
+Links
+-----
+
+See our ICLR Tiny Paper: https://arxiv.org/pdf/2304.04000.pdf
```
Comment (MaximilianKleissl) [inline docs/source/index.rst:43]: This links to Arxiv and not to ICLR.

Comment (jzbbrv-hpi) [reply docs/source/index.rst:43]: Very true, the ICLR version is still in the making. So this is a tempary link. Should I put "on Arxiv" at the end of the sentence?

Comment (MaximilianKleissl) [reply docs/source/index.rst:56]: See cite us:
https://github.com/DILiS-lab/SimbaML/blob/main/docs/source/about/cite.rst

Comment (MaximilianKleissl) [reply docs/source/index.rst:56]: By the way: If you want, you can change Submitted to Accepted in that file.

Comment (jzbbrv-hpi) [reply docs/source/index.rst]: Should be fixed in the latest commit.

**Pull request merged**